app/views/liquids/_open_graph.html.erb

```diff
@@ -0,0 +1,33 @@
+<% if page.main_properties_present? %>
+  <div class="crayons-card p-5">
+    <% if page.image %>
+      <div class="s:max-w-50 mb-4">
+        <img src="<%= page.image %>" height="auto" alt="Cover image">
```

Comment:
```suggestion
        <img src="<%= page.image %>" height="auto" alt="">
```
Let's leave this blank, as "Cover image" (or anything we provide) can't really add any value for screen reader users. If we leave the alt blank we're basically saying "this image is only decorative" which I think is the best that we can do given we don't know what the content is 🙂